terraform/aws/cognito.tf

```
@@ -34,7 +34,7 @@ resource "aws_cognito_user_pool_client" "atlas_pool_client" {
   user_pool_id = aws_cognito_user_pool.atlas_pool.id
   callback_urls = concat(
     ["https://api.${var.domain_name}/docs/oauth2-redirect"],
-    jsondecode(var.cors_origins),
+    jsondecode(var.cognito_redirect_urls),
   )
```

```suggestion
  callback_urls = jsondecode(var.cognito_redirect_urls)
```
Let's just pull the `api` one in as well to simplify

---

Made this change